Sources/ServiceLifecycle/ServiceRunnerError.swift

```diff
@@ -50,14 +50,16 @@ public struct ServiceGroupError: Error, Hashable, Sendable {
 
     /// Internal class that contains the actual error code.
     private final class Backing: Hashable, Sendable {
+        let message: String?
         let errorCode: Code
         let file: String
         let line: Int
 
-        init(errorCode: Code, file: String, line: Int) {
+        init(errorCode: Code, file: String, line: Int, message: String? = nil) {
```

Comment:
```suggestion
        init(errorCode: Code, file: String, line: Int, message: String?) {
```

Nit: I'd drop the default